app/pulp/app/serializers/repository.py

```diff
@@ -36,11 +37,18 @@ class RepositorySerializer(ModelSerializer):
     )
     notes = NotesKeyValueRelatedField()
     scratchpad = ScratchpadKeyValueRelatedField()
+    importers = ImporterRelatedField(many=True)
```

Comment:
My understanding is that the reasoning for calling this `importers` and `that many=True` is to allow forward compatibility for multiple importers per repository. If so, a comment seems warranted.